host-controller/src/main/java/org/jboss/as/host/controller/DomainModelControllerService.java

```diff
@@ -897,11 +897,22 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                     Notification notification = new Notification(ModelDescriptionConstants.BOOT_COMPLETE_NOTIFICATION, PathAddress.pathAddress(PathElement.pathElement(CORE_SERVICE, MANAGEMENT),
                             PathElement.pathElement(SERVICE, MANAGEMENT_OPERATIONS)), ControllerLogger.MGMT_OP_LOGGER.bootComplete());
                     getNotificationSupport().emit(notification);
-                    bootstrapListener.printBootStatistics();
+                    List<String> configFiles = new ArrayList<String>();
```

Comment:
Minor detail, you can use Java Type inference capabilities, so instead of `new ArrayList<String>()` use `new ArrayList<>()` here.